test/monitoring/aws-secretsmanager/__snapshots__/SecretsManagerSecretMonitoring.test.ts.snap

```diff
@@ -2,6 +2,130 @@
 
 exports[`each stack in an app gets its own publisher instance 1`] = `
 Object {
+  "Mappings": Object {
```

Comment:
I feel like the snapshot should be able to optimize out the regions that aren't used 😩 ..this is just bloat for our snaps

Reply:
I can probably build on top of https://github.com/cdklabs/cdk-monitoring-constructs/pull/595 to remove these.

Reply:
Now explicitly removed from snapshots.